samples/ordered_puzzle/ordered_puzzle.html

```diff
@@ -30,11 +30,12 @@
         ></a-entity>
         <a-entity
           id="ordered-puzzle"
-          ordered-puzzle='jsonData: {"images": [{"imageSrc" : "./images/pic1.png", "width": 3, "height": 3, "xTarget": -7, "yTarget": 0, "x": 0, "y": 4},
-          {"imageSrc" : "./images/pic2.png", "width": 3, "height": 3, "xTarget": -3.5, "yTarget": 0, "x": -2, "y": -4},
-          {"imageSrc" : "./images/pic3.png", "width": 3, "height": 3, "xTarget": 0, "yTarget": 0, "x": -4, "y": 4},
-          {"imageSrc" : "./images/pic4.png", "width": 3, "height": 3, "xTarget": 3.5, "yTarget": 0, "x": 2, "y": -4},
-          {"imageSrc" : "./images/pic5.png", "width": 3, "height": 3, "xTarget": 7, "yTarget": 0, "x": 4, "y": 4}]}'
+          ordered-puzzle='jsonData: {"images": [{"imageSrc" : "./images/pic1.png", "width": 3, "height": 3, "xTarget": -7, "yTarget": 0},
+          {"imageSrc" : "./images/pic2.png", "width": 3, "height": 3, "xTarget": -3.5, "yTarget": 0},
+          {"imageSrc" : "./images/pic3.png", "width": 3, "height": 3, "xTarget": 0, "yTarget": 0},
+          {"imageSrc" : "./images/pic4.png", "width": 3, "height": 3, "xTarget": 3.5, "yTarget": 0},
+          {"imageSrc" : "./images/pic5.png", "width": 3, "height": 3, "xTarget": 7, "yTarget": 0}],
+          "useTargets": false}'
```

Comment:
Are we able to call this without the `xTarget` and `yTarget` fields if `useTargets` is false? Ideally we only need to include them if useTargets is true (ie. no errors if not included)

Reply:
Yes, xTarget and yTarget fields don't need to be provided if useTargets is false.